intro-OpenStack/ch_LaunchInstances.tex

```diff
@@ -0,0 +1,429 @@
+recommended), you do not need to provide a key pair to launch the
+instance.
+
+Configuration tab
+
+\textbf{Customization Script Source}
+
+Specify a customization script that runs after your instance launches.
+
+Metadata tab
+
+\textbf{Available Metadata}
+
+Add Metadata items to your instance.
+\end{quote}
+
+\begin{enumerate}
+\def\labelenumi{\arabic{enumi}.}
+\item
+  \begin{quote}
+  Click Launch Instance.
+  \end{quote}
+\end{enumerate}
+
+\begin{quote}
+The instance starts on a compute node in the cloud.
+\end{quote}
+
+~
+
+Note
+
+If you did not provide a key pair, security groups, or rules, users can
+access the instance only from inside the cloud through VNC. Even pinging
+the instance is not possible without an ICMP rule configured.
+
+You can also launch an instance from the Images or Volumes category when
+you launch an instance from an image or a volume respectively.
+
+When you launch an instance from an image, OpenStack creates a local
+copy of the image on the compute node where the instance starts.
+
+For details on creating images, see
+\href{https://docs.openstack.org/image-guide/create-images-manually.html}{\emph{Creating
+images manually}} in the \emph{OpenStack Virtual Machine Image Guide}.
+
+When you launch an instance from a volume, note the following steps:
+
+\begin{itemize}
+\item
+  \begin{quote}
+  To select the volume from which to launch, launch an instance from an
+  arbitrary image on the volume. The arbitrary image that you select
+  does not boot. Instead, it is replaced by the image on the volume that
+  you choose in the next steps.
+  \end{quote}
+\end{itemize}
+
+\begin{quote}
+To boot a Xen image from a volume, the image you launch in must be the
+same type, fully virtualized or paravirtualized, as the one on the
+volume.
+\end{quote}
+
+\begin{itemize}
+\item
+  \begin{quote}
+  Select the volume or volume snapshot from which to boot. Enter a
+  device name. Enter \textbf{vda} for KVM images or \textbf{xvda} for
+  Xen images.
+  \end{quote}
+\end{itemize}
+
+~
+
+Note
+
+When running QEMU without support for the hardware virtualization, set
+\textbf{cpu\_mode="none"} alongside \textbf{virt\_type=qemu} in
+\textbf{/etc/nova/nova-compute.conf} to solve the following error:
+
+libvirtError: unsupported configuration: CPU mode 'host-model'
+
+for ``x86\_64`` qemu domain on ``x86\_64`` host is not supported by
+hypervisor
+
+\strong{Connect to your instance by using SSH}\label{connect-to-your-instance-by-using-ssh}
+
+To use SSH to connect to your instance, use the downloaded keypair file.
+
+~
+
+Note
+
+The user name is \textbf{ubuntu} for the Ubuntu cloud images on
+TryStack.
+
+\begin{enumerate}
+\def\labelenumi{\arabic{enumi}.}
+\item
+  \begin{quote}
+  Copy the IP address for your instance.
+  \end{quote}
+\item
+  \begin{quote}
+  Use the \textbf{ssh} command to make a secure connection to the
+  instance. For example:
+  \end{quote}
+\item
+  \begin{quote}
+  \$ ssh -i MyKey.pem ubuntu@10.0.0.2
```

Comment:
connecting to the loginnode is the default forewall policy.
when people ask for a public ip, they have 2 options:
* do nothing, and use the ssh proxy through the loginnodes. this should work immediately
* contact cloud@vscentrum.bne and ask to modify the firewall for their public ip so it can be reached from some subnet/theworld/.... this however might take some time, but after that, should be fine.

they still might need to setup ssh proxy to eg ssh to the vms that are behind a jumphost or something like that; but that does not change anything.

Reply:
about the ssh agent in windows, this info is included in #330 for intro-HPC

Reply:
> connecting to the loginnode is the default firewall policy.
> when people ask for a public ip, they have 2 options:
> 
>     * do nothing, and use the ssh proxy through the loginnodes. this should work immediately

How does this ssh proxy work, exactly?  Or do you have to set up such proxying on demand?